pkg/gocachemanager/sdk.go

```diff
@@ -68,6 +73,20 @@ func NewGoCacheWrapper(
 		caches = append(caches, cache.New[string](redisStore))
 	}
 
+	if settings.redisClusterConnection != "" {
+		redisClusterClient := redis.NewClusterClient(&redis.ClusterOptions{
+			Addrs: strings.Split(settings.redisClusterConnection, ","),
+		})
```

Comment:
Prefer redis.ParseClusterOptions(settings.redisClusterConnection) instead of splitting.